### PR TITLE
Fix some typos and inconsistencies in documentation

### PR DESCRIPTION
**Pull request by agraboso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,7 +366,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
-  return bindActionCreators(Object.assign({}, todoActionCreators, counterActionCreators), dispatch);
+  return bindActionCreators({ ...todoActionCreators, ...counterActionCreators}, dispatch);
```
Comment (gaearon) [inline README.md]: We'd rather not use ES7 experimental code in examples to avoid confusing beginners.

Comment (agraboso) [inline README.md]: There are several other instances of object spread notation in the README, both in the text and in the examples. If you want, I can revert them all back to Object.assign.

Comment (agraboso) [inline README.md]: Hmm, in fact there were only the two that I introduced and two more.

Comment (gaearon) [inline README.md]: Yeah, I'd prefer to use `assign` everywhere.

**Comment by gaearon** [comment]
Thanks.

**Pull request merged**